scripts/recalc_abilities_for_all_users.rb

```diff
@@ -0,0 +1,18 @@
+puts "Recalculating all the Abilities"
+puts
+
+User.unscoped.all.map do |u|
+    puts "Scope: User.Id=#{u.id}"

```

Comment:
Rubocop's not going to like your 4-space indents...

Reply:
Luckily RC doesn’t scan that file. :)